### ISSUE TITLE
replace url for /data-library

### ISSUE THREAD

**Issue by matallo** [CartoDB/cartodb#6041]
make it hang from `/data` so it finally is `/data/data-library` and `/data/datasets/:name`

**Comment by matallo** [comment]
still blocked by https://github.com/CartoDB/cartodb-platform/issues/923

**Comment by matallo** [comment]
does this still apply @saleiva ?

**Comment by xavijam** [comment]
There has been no activity on this issue for more several months. We are closing it. If you think this still needs to be addressed please open a new issue.
